components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -347,7 +347,7 @@ data class ContextMenuCandidate(
         ) = ContextMenuCandidate(
             id = "mozac.feature.contextmenu.copy_link",
             label = context.getString(R.string.mozac_feature_contextmenu_copy_link),
-            showFor = { _, hitResult -> hitResult.isLink() },
+            showFor = { _, _ -> true },
```

Comment:
I don't think this makes sense: This will show the context menu item for everything you long press, even if there is no link at all. You may need to change `isLink()` since that checks for `http`: https://github.com/mozilla-mobile/android-components/blob/master/components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt#L432

CC @jonalmeida

Reply:
The current code checks if the link _starts with_ http, so it should work with https. Maybe the issue is elsewhere?

Reply:
Ooh weird, isLink only works on IMAGE_SRC hit results. Sorry, I must have missed that before.

Reply:
The main issue here is that we need this to work with more than http/https.  It should work with any type link.